File: WEKEE.BE/Account.Domain/Shared/DataTransfer/ResourceDTO/ResourceReadDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Account.Domain.Shared.DataTransfer.ResourceDTO
{
    public class ResourceReadDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int TypesRsc { get; set; }
        public string Description { get; set; }
        public bool? IsActive { get; set; }
        public int CreateBy { get; set; }
        public string CreateByName { get; set; }
        public DateTime CreatedOnUtc { get; set; }
        public DateTime UpdatedOnUtc { get; set; }
    }
}

File: WEKEE.FE/UI.HOME/src/components/AdminBasic/dto/resourceReadDto.ts
export interface ResourceReadDto {
    id: number;
    name: string;
    typesRsc: number;
    description: string;
    isActive: boolean | null;
    createBy: number;
    createByName: string;
    createdOnUtc: string;
    updatedOnUtc: string;
}